src/style.css

```diff
@@ -1,3 +1,78 @@
 body {
+  font-family: Roboto, system-ui;
```

Comment:
It's recommended to use a consistent vertical rhythm in your CSS. Instead of setting the font-size here, consider using a CSS variable at the root level and apply it where needed to maintain consistency.